docs/conf.py

```diff
@@ -0,0 +1,178 @@
+# -*- coding: utf-8 -*-
+#
+# Configuration file for the Sphinx documentation builder.
+#
+# This file does only contain a selection of the most common options. For a
+# full list see the documentation:
+# http://www.sphinx-doc.org/en/master/config
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'Atlassian Python API'
+copyright = '2018, SLRover'
```

Comment:
Hi! 
Thanks for awesome docs provides.
Can we set the Apache 2 license info? 
@AstroMatt  When we migrate into project, can we set the organisation copiright? 

Cheers,
Gonchik Tsymzhitov

Reply:
Yes. Let's migrate to the organization!
https://github.com/atlassian-api

This will be a new place for a project.